src/jit/block.h

```diff
@@ -743,16 +743,14 @@ struct BasicBlock : private LIR::Range
         return bbRefs;
     }
 
-    __declspec(property(get = getBBTreeList, put = setBBTreeList)) GenTree* bbTreeList; // the body of the block.
-
     Statement* bbStmtList;
 
-    GenTree* getBBTreeList() const
+    GenTree* GetBBTreeList() const
```

Comment:
I now realize that this mechanical replacement of the `declspec` crap might turn out to be a bit more messier than one would like. If we spend time doing this then maybe we should try to come up with better names when necessary - like here where something like `GetFirstLIRNode` would be preferable to this nondescript `GetBBTreeList`.

On the other hand if we debate the name of every declspec that needs to be removed we probably won't finish this before .NET 5 :grin:. Oh well.

Reply:
I don't mind doing a search and replace if you have a better suggestion. There is a total of 75 changes that need to happen. As long as we close the changes in timely manner, it is fine.

I'll apply your suggestion.

Reply:
Thanks, looks pretty good.